### PR TITLE
Multimodal Textbox Loading + other fixes

### PR DESCRIPTION
**Pull request by dawoodkhan82**
## Description

Replaces "send" emoji icon with svg.
Removed `UploadProgress` and added a progress loading animation
+some other small css fixes

Screenshot 2024-03-26 at 4 56 55 PM

Closes: #7768
Closes: #7812

## 🎯 PRs Should Target Issues

Before your create a PR, please check to see if there is [an existing issue](https://github.com/gradio-app/gradio/issues) for this change. If not, please create an issue before you create this PR, unless the fix is very small. 

Not adhering to this guideline will result in the PR being closed. 

## Tests

1. PRs will only be merged if tests pass on CI. To run the tests locally, please set up [your Gradio environment locally](https://github.com/gradio-app/gradio/blob/main/CONTRIBUTING.md) and run the tests: `bash scripts/run_all_tests.sh`

2. You may need to run the linters: `bash scripts/format_backend.sh` and `bash scripts/format_frontend.sh`

### CODE REVIEW

File: gradio/components/multimodal_textbox.py [hunk 1]
```diff
@@ -74,7 +74,7 @@ def __init__(
         render: bool = True,
         text_align: Literal["left", "right"] | None = None,
         rtl: bool = False,
-        submit_btn: str | Literal[False] = "⌲",
+        submit_btn: str | Literal[False] = "",
```
Comment (abidlabs) [inline gradio/components/multimodal_textbox.py:77]: Better to use `None` I think? And we should mention in the docstring that if None, an svg of a send icon is used.
```suggestion
        submit_btn: str | Literal[False] | None = None,
```

**Review by abidlabs — APPROVED** [review]
Beautiful! Looks great. Just left one suggestion on the python side

**Review by pngwn — APPROVED** [review]
Looks good! Thanks for taking care of this @dawoodkhan82 !

**Comment by abidlabs** [comment]
Ok great, let me go ahead and merge this in so that we can get the release out!

**Pull request merged**